core/src/main/java/com/redhat/lightblue/client/response/LightblueBulkResponseException.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.lightblue.client.response;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class LightblueBulkResponseException extends LightblueException {
+
+    private static final long serialVersionUID = -1204842853642316889L;
+
+    private final LightblueBulkDataResponse bulkResponse;
+    private final Map<Integer, LightblueResponseException> erroredResponses;
+
+    public LightblueBulkResponseException(String message,
+            LightblueBulkDataResponse bulkResponse,
+            Map<Integer, LightblueResponseException> erroredResponses) {
+        super(message);
+        this.bulkResponse = bulkResponse;
+        this.erroredResponses = erroredResponses;
+    }
+
+    public LightblueBulkDataResponse getBulkResponse() {
+        return bulkResponse;
```

Comment:
I did this to be consistent with how LightblueResponseException works.

That said, I think a unit test showing how to get just the successful responses might be useful. Then, maybe it could be made easier.